content/docs/next-release/deployment.md

```diff
@@ -146,6 +146,19 @@ data will be lost once the process is gone.
 By default, there's no limit in the amount of traces stored in memory but a limit can be established by passing an
 integer value via `--memory.max-traces`.
 
+### Badger - local storage
+Experimental since Jaeger 1.9
+
+[Badger](https://github.com/dgraph-io/badger) is an embedded local storage available in all-in-one distribution.
+By default it acts as an ephemeral storage using temporary filesystem which can be overridden by `--badger.ephemeral=false`.
```

Comment:
Slight reword: "By default it acts as an ephemeral storage using a temporary filesystem. This can be overridden by using the `--badger.ephemeral=false` option.